decidim-core/app/types/decidim/core/user_type.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Core
+    # This type represents a User.
+    UserType = GraphQL::ObjectType.define do
+      name "User"
+      description "A user"
+
+      interfaces [
+        Decidim::Core::AuthorInterface
+      ]
+
+      field :name, !types.String, "The user's name"
+
+      field :nickname, !types.String, "The user's nickname" do
+        resolve ->(obj, _args, _ctx) { UserPresenter.new(obj).nickname }
+      end
+
+      field :avatarUrl, !types.String, "The user's avatar url" do
+        resolve ->(obj, _args, _ctx) { UserPresenter.new(obj).avatar_url(:thumb) }
+      end
+
+      field :profilePath, !types.String, "The user's profile url" do
+        resolve ->(obj, _args, _ctx) { UserPresenter.new(obj).profile_path }
+      end
+
+      field :organizationName, !types.String, "The user's organization name" do
```

Comment:
Does this mean the API is returning data for *all* the organizations?

Reply:
No - AFAIK this is used to interpolate the name of the organization when showing a locale in comments: 

https://github.com/decidim/decidim/blob/40c601a7ff8a0fa0e53434089f88c0b780b2540a/decidim-comments/app/frontend/comments/comment.component.tsx#L424